### PR TITLE
Price changes for default pages

### PR DESCRIPTION
**Pull request by george-icarcover**
Premium Plus (PP)
Car - $159.95
SUV - $179.95
Truck - $179.95

Premium (PS)
Car - $119.95
SUV - $139.95
Truck - $159.95

Standard Pro (SP)
Car - $100
SUV - $120
Truck - $140

Standard (SS)
Car - $80
SUV - $100
Truck - $120

### CODE REVIEW

File: src/app/(main)/[productType]/components/CarCoverSelector.tsx [hunk 1]
```diff
@@ -53,8 +53,6 @@ export function CarCoverSelector({
 
   const productImages = selectedProduct?.productImages as string[];
 
-  const productName = modelData[0]?.fullProductName;
```
Comment (devwilliamy) [inline src/app/(main)/[productType]/components/CarCoverSelector.tsx:56]: Why did you get rid of this? Is it not being used?

**Review by devwilliamy — APPROVED** [review]
Looks good for the most part, please have Calvin test it out on your branch before pushing to preview

File: src/app/(main)/[productType]/components/CarCoverSelector.tsx (hunk 1 above)
Comment (george-icarcover) [reply src/app/(main)/[productType]/components/CarCoverSelector.tsx:56]: ESLint said it was going unused so I removed.

File: src/app/(main)/[productType]/components/ProductContent.tsx [hunk 2]
```diff
@@ -58,49 +58,66 @@ export function ProductContent({
...
   console.log('Model Data: ', modelData);
   console.log('Selected Product: ', selectedProduct);
 
-  let installmentPrice: number;
-  switch (coverType) {
-    case 'premium':
-      installmentPrice = 29.99;
+  let defaultMSRP: number;
+
+  switch (true) {
```
Comment (devwilliamy) [inline src/app/(main)/[productType]/components/ProductContent.tsx:79]: Generally, you won't use a switch case like this. If you're doing it like this, you'd just use an if/else

There's also a few cases you're able to combine, namely if the defaultMSRP is the same

**Pull request merged**